admin/tools/docker/deployment/kubernetes/admin/mkdir.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#  Create directory belonging to a given user
+#  on all hosts
+
+# @author Fabrice Jammes SLAC/IN2P3
+
+set -e
+set -x
+
+DIR=$(cd "$(dirname "$0")"; pwd -P)
+. "$DIR/../env-cluster.sh"
+
+REMOTE_DIR="/qserv/data"
+echo "Create directory $REMOTE_DIR on all nodes"
+
+REMOTE_USER=centos
```

Comment:
This only works on CentOS?

Reply:
This should be parametrized in the long term, but our only target system is centos for now.